Source/UI/Screens/RequiredFields/RequiredFieldsViewController.swift

```diff
@@ -231,6 +233,9 @@ class RequiredFieldsViewController: IdentityUIViewController {
     }
 }
 
+extension RequiredFieldsViewController: UITextViewDelegate {
```

Comment:
forgot to delete extension or forgot to add members :p

Reply:
I added it as a placeholder since it'll be needed to handle links, and by having it I could already set the view controller as the text view's delegate. Too far maybe? 😛

Reply:
Haha maybe a bit too eager 😂 I'll leave this to you to merge and add to in a follow up PR then :)